src/Database.java

```diff
@@ -0,0 +1,97 @@
+import java.sql.*;
+import java.util.Map;
+import java.util.Scanner;
+
+public class Database {
+    // Connect to your database
+    public static void main(String[] args) throws SQLException {
+        Map<String, String> env = System.getenv();
+        char[] alphabet = {
+                'A', 'B', 'C', 'D', 'E', 'F', 'G',
+                'H', 'I', 'J', 'K', 'L', 'M', 'N',
+                'O', 'P', 'Q', 'R', 'S', 'T', 'U',
+                'V', 'W', 'X', 'Y', 'Z'
+        };
+
+        String endpoint = "database-1.ckxf3a0k0vuw.us-east-1.rds.amazonaws.com";
+
+        String connectionUrl =
+                "jdbc:sqlserver://" + endpoint + ";"
+                        + "database=YaacovStuhl;"
+                        + "user=MCON364;"
+                        + "password=Pesach2025;"
+                        + "encrypt=true;"
+                        + "trustServerCertificate=true;"
+                        + "loginTimeout=30;";
```

Comment:
credentials should NEVER be in source code! huge security risk. use properties file or env variables